src/components/FlowRunFilteredList.vue

```diff
@@ -19,7 +19,7 @@
       </template>
     </p-list-header>
 
-    <FlowRunList v-model:selected="selected" :flow-runs="flowRuns" :selectable="selectable && can.delete.flow_run" @bottom="next" />
+    <FlowRunList v-model:selected="selected" v-bind="{ hideDetails, hideFlowName }" :flow-runs="flowRuns" :selectable="selectable && can.delete.flow_run" @bottom="next" />
```

Comment:
This component isn't being used on the dashboard but is used on the flow page where we can remove the flow name since that's a known context

Reply:
Its being used in quite a few places. But the defaults are false so we have to intentionally hide things. So this seems fine to me

Reply:
Yeah that was the intent - this won't change anything but the dash by default